experimental/c-cpp/manager.py

```diff
@@ -137,12 +137,16 @@ def add_to_source_cache(heuristic_name, target_func, fuzzer_source):
 
 class FuzzHeuristicGeneratorBase:
   """Base class for fuzzer heuristics generator."""
+  language = ''
+  name = ''
```

Comment:
nit: Would it be better to initialize them in `__ini__()`?

Reply:
Not in this case because we query the language before initializing an object of the class.